configs/enable-dvo-for-all-cluster/install-dvo.yaml

```diff
@@ -20,12 +20,13 @@ metadata:
   name: deployment-validation-operator
   labels:
     name: deployment-validation-operator
+  namespace: deployment-validation-operator
 spec:
   channel: alpha
   installPlanApproval: Automatic
   name: deployment-validation-operator
   source: community-operators
-  sourceNamespace: deployment-validation-operator
+  sourceNamespace: openshift-marketplace
```

Comment:
if we use community-operators catalog, do we still need to define our own catalog source(line 1 to 15)?

Reply:
I tested on my dev cluster without the custom catalog source and the operator installed without any issue.

Reply:
done